src/components/backend-ai-credential-view.ts

```diff
@@ -753,27 +760,27 @@ export default class BackendAICredentialView extends BackendAIPage {
       return value % 1 ? value.toString().split('.')[1].length : 0;
     };
 
+    if (textfield.value === '') {
+      textfield.value = textfield.min ?? 0;
+    }
+
     if (textfield.classList.contains('discrete')) {
       textfield.value = Math.round(textfield.value);
     }
 
     if (textfield.value <= 0) {
       // concurrency job and container-per-session limit must be upper than 0.
```

Comment:
I'm not sure this statement is true, but it is the policy set before this PR, so let's keep it intact. And, "upper" should be changed to "greater" or "larger".

Reply:
I agree. But I'm afraid we need to remove those statements since it doesn't do anything.

Reply:
> Also, the capacity does not seem to be updated correctly.
> 
>  2022-07-06.23.36.52.mov

I confirmed the bug happened with the same approach. I just fixed it.